tests/__init__.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
```

Comment:
Is there are reason to add shebang here?

Reply:
Nope, removed.